srcpkgs/runit-void/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'runit-void'
 pkgname=runit-void
 version=20231124
-revision=5
+revision=6
```

Comment:
please rebase (revision should go to 7 because of 6067b38ec7f54d949290e5c98915dad812d097b1)